Sources/Jobs/JobsCommand.swift

```diff
@@ -119,18 +119,21 @@ public class JobsCommand: Command {
                 console.info("Dequeing Job job_id=[\(jobStorage.id)]", newLine: true)
                 
                 let futureJob = job.anyDequeue(jobContext, jobStorage)
-                return self.firstFutureToSucceed(future: futureJob, tries: jobStorage.maxRetryCount, on: eventLoop).flatMap { _ in
```

Comment:
I think it might be better here to create a new promise and explicitly use `.do/.catch/.always` to handle the possible cases. I think that would make the error handling a lot clearer.